test/commands/force/apex/test/run.test.ts

```diff
@@ -1229,21 +989,10 @@ describe('force:apex:test:run', () => {
     .stub(TestService.prototype, 'runTestAsynchronous', () => testRunSimple)
     .stdout()
     .stderr()
-    .command([
-      'force:apex:test:run',
-      '--tests',
-      'MyApexClass.testInsertTrigger',
-      '--outputdir',
-      'my/path/to/dir',
-      '-r',
-      'human'
-    ])
-    .it(
-      'should display warning message when output directory flag is specifed',
-      ctx => {
-        expect(ctx.stderr).to.include(messages.getMessage('warningMessage'));
-      }
-    );
+    .command(['force:apex:test:run', '--tests', 'MyApexClass.testInsertTrigger', '--outputdir', outDir, '-r', 'human'])
```

Comment:
Replacing 'my/path/to/dir' with outDir here is the change that was made to get the Windows unit tests passing.  This is changed in 3 places.  The rest of the changes in this file are prettier formatting changes.